kahuna/public/js/components/gr-title/gr-title.js

```diff
@@ -0,0 +1,20 @@
+import angular from 'angular';
+import 'titip';
+
+export const module = angular.module('grTitle', []);
+
+module.directive('grTitle', [function () {
```

Comment:
> Would "tooltip" be a little more obvious than "title"?

I did that initially, but opted for grTitle as its essentially a custom styled `title` attribute.

> can use these for more than just snazzy titles

Yep! Was planning on creating a `grRichTitle` directive, which, thinking about it, isn't the best name, so maybe `tooltip` is best!